File: src/main/java/com/sandbox/Application.java
package com.sandbox;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import reactor.blockhound.BlockHound;

@SpringBootApplication
class Application {
    public static void main(String... args) {
        BlockHound.install();
        SpringApplication.run(Application.class, args);
    }
}

File: src/main/kotlin/com/sandbox/Application.kt
package com.sandbox

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import reactor.blockhound.BlockHound

@SpringBootApplication
class Application {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            BlockHound.install()
            SpringApplication.run(Application::class.java, *args)
        }
    }
}